core/src/main/scala/stryker4s/run/ProcessMutantRunner.scala

```diff
@@ -65,9 +65,8 @@ class ProcessMutantRunner(command: Command, process: ProcessRunner)(implicit con
     MutantRunResults(runResults, calculateMutationScore(totalMutants, detected.size), duration)
   }
 
-  private[this] def runMutant(mutant: Mutant, workingDir: File, subPath: Path): MutantRunResult = {
-    val id = mutant.id
-    info(s"Starting test-run ${id + 1}...")
+  private[this] def runMutant(mutant: Mutant, workingDir: File, subPath: Path, id: Int): MutantRunResult = {
+    info(s"Starting test-run $id...")
     process(command, workingDir, ("ACTIVE_MUTATION", id.toString)) match {
```

Comment:
You could pass in the index of the mutation so the log statement is correct. but you will still have to use mutant.id to run the process otherwise you will have the wrong active mutant.